src/OpenTelemetry.Instrumentation.SqlClient/SqlClientTraceInstrumentationOptions.cs

```diff
@@ -255,19 +282,19 @@ internal static SqlConnectionDetails ParseDataSource(string dataSource)
 
     internal void AddConnectionLevelDetailsToActivity(string dataSource, Activity sqlActivity)
     {
-        if (!this.EnableConnectionLevelAttributes)
-        {
-            sqlActivity.SetTag(SemanticConventions.AttributePeerService, dataSource);
-        }
```

Comment:
This instrumentation should not be emitting `peer.service`.

But just a reminder here... when `peer.service` was removed from HTTP client instrumentation it was unexpected see #1761.